grails-app/services/org/pih/warehouse/invoice/PrepaymentInvoiceService.groovy

```diff
@@ -197,18 +197,22 @@ class PrepaymentInvoiceService {
         if (!prepaymentItem) {
             return null
         }
-        if (orderAdjustment.inversedQuantity > 0) {
+
+        BigDecimal amountToInverse = getAmountAvailableToInverse(orderAdjustment, prepaymentItem)
+        // TODO: what if canceled and no inverse yet? Should work, but have to check that case.
+        // TODO: plus check cancel + uncancel
```

Comment:
Did you create a ticket for that? If not, when it should be checked?

Reply:
As this is still in WIP, these are TODOs that I sill have to check within this ticket. These are reminders to myself.